spark_client/services.py

```diff
@@ -464,19 +937,22 @@ def set_primary(self, account_id: str) -> str:
         """
 
         # Relabeling primary
-        primary_account = self.get_primary()
+        try:
+            primary_account = self.get_primary()
```

Comment:
**todo** I believe this does not comply with the signature. The method is supposed to return None if there is no primary. There was a bug in the code, that I have corrected in the PR #45